src/FiberTraces.cc

```diff
@@ -1047,6 +1047,8 @@ namespace pfsDRPStella = pfs::drp::stella;
       throw LSST_EXCEPT(pexExcept::Exception, message.c_str());
     }
 
+    double tmpVal;
```

Comment:
Moved the declaration down to just before the `for` loops where it will be used, and initialized it. I don't want to move the declaration inside the `for` loops as it would cause multiple memory allocations and deallocations.